src/hooks/prerun/install-prerun.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-console */
+const chalk = require('chalk');
+
+const UPDATE_ALERT = `Warning: Use '${chalk.bold('npm update twilio-cli')}' for npm based installations. `;
```

Comment:
Adding a new prerun hook to warn users about the update feature not working for npm based installations: oclif/plugin-update#49